webapp/src/components/sidebar_right/index.tsx

```diff
@@ -51,23 +51,6 @@ export const renderThumbVertical = (props: Props) => (
     />
 );
 
-function shouldUpdateDetails(prs: Item[], prevPrs: Item[], targetState: string, currentState: string) {
-    if (currentState !== targetState) {
-        return false;
-    }
-
-    if (prs.length !== prevPrs.length) {
-        return true;
-    }
-
-    for (let i = 0; i < prs.length; i++) {
-        if (prs[i].id !== prevPrs[i].id) {
-            return true;
-        }
-    }
-    return false;
-}
-
 function mapPrsToDetails(prs: Item[], details: Item[]) {
```

Comment:
There is a comment on the MM PR to do these computations in a reducer or in a selector, but that change is not present in this PR.

Reply:
yes, it stated to avoid an extra state, and do the computations in redux or selector. But we need that state to store the previous details of PRs and reviews.

Reply:
Please add appropriate replies to all the questions on that PR.